tests/unit/src/test/scala/tests/NewFileLspSuite.scala

```diff
@@ -132,8 +132,7 @@ class NewFileLspSuite extends BaseLspSuite("new-file") {
     expectedFilePath = "a/src/main/scala/foo/Foo.scala",
     expectedContent = """|package foo
                          |
-                         |final case class Foo()
-                         |""".stripMargin
+                         |final case class Foo()""".stripMargin
```

Comment:
Why did this test change actually? Shouldn't we have a newline after the definition?

Reply:
fixed it too. there were no newline